app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/InputWidgets/index.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { ChangeEvent } from 'react';
+import { IInputWidgetsProps } from 'components/WranglerGrid/SelectColumnPanel/DataTable/types';
+import { Checkbox, FormControlLabel, Radio } from '@material-ui/core';
+import styled from 'styled-components';
+
+const RadioInput = styled(Radio)`
+  &.MuiRadio-root {
+    padding: 0;
+  }
+`;
+
+export default function({
+  isSingleSelection,
+  selectedColumns,
+  handleSingleSelection,
+  columnDetail,
+  handleDisableCheckbox,
+  handleMultipleSelection,
+  columnIndex,
+}: IInputWidgetsProps) {
+  const getDisableAttributeValue = () =>
+    selectedColumns.findIndex((column) => column.label === columnDetail.label) > -1 ||
+    !handleDisableCheckbox()
```

Comment:
why are you returning booleans here and below? Also it doesn't seem like handleDisableCheckbox actually disables the checkbox, it returns a boolean so this should be something like isCheckboxDisabled or something like that